CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ Every entry has a category for which we use the following visual abbreviations:
 
 ## Unreleased
 
+- 🎁 We open-sourced our [MessagePack](http://msgpack.org)-based Table Slice
+  implementation. `msgpack` is now a valid option for the
+  `import.table-slice-type` configuration option, and is the new default when
+  Apache Arrow is unavailable. We recommend using MessagePack for row-bsaed
+  formats like PCAP. [#975](https://github.com/tenzir/vast/pull/975)
```

Comment:
```suggestion
- 🎁 We open-sourced our [MessagePack](http://msgpack.org)-based table slice
  implementation, which provides a compact row-oriented encoding of data. This
  encoding works well for binary formats (e.g., PCAP) and access patterns that
  involve materializing entire rows. The MessagePack table slice is the new
  default when Apache Arrow is unsavailabe. To enable parsing into MsgPack,
  you can pass `--table-slice-type=msgpack` to `vast import` or set the
  configuration option `import.table-slice-type` to msgpack. 
  [#975](https://github.com/tenzir/vast/pull/975)
```

Reply:
I went with a slight variation for consistency and spelling fixes.